src/quafel/pipelines/data_science/frameworks.py

```diff
@@ -162,6 +163,28 @@ def get_result(self) -> Dict[str, float]:
 #         return counts
 
 
+class qrisp_fw:
+    def __init__(self, qasm_circuit, n_shots):
+        self.n_qubits = calculate_n_qubits_from_qasm(qasm_circuit)
+
+        self.qc = qrisp.QuantumCircuit.from_qasm_str(qasm_circuit)
+        self.n_shots = n_shots
+        self.result = None
+
+    def execute(self) -> None:
+        self.result = self.qc.run(shots=self.n_shots)
+
+    def get_result(self) -> Dict[str, float]:
+        counts = self.result
+
+        for i in range(2**self.n_qubits):
+            bitstring = format(i, f"0{self.n_qubits}b")
+            if bitstring not in counts.keys():
+                counts[bitstring] = 0
```

Comment:
In case you are using this for several of the frameworks you might consider refactoring this as a function that takes care on filling up the blanks in your counts dictionary.

Reply:
good point, will create an issue and refactor this in another pr as it affects multiple fws